estate/views/estate_property_views.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="estate_property_action" model="ir.actions.act_window">
+        <field name="name">Properties</field>
+        <field name="res_model">estate.property</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+
+    <record id="estate_property_view_tree" model="ir.ui.view">
+        <field name="name">estate_property_tree</field>
+        <field name="model">estate.property</field>
+        <field name="arch" type="xml">
+            <tree string="Estate List View">
+                <field name="name"/>
+                <field name="postcode"/>
+                <field name="bedrooms"/>
+                <field name="living_area"/>
+                <field name="expected_price"/>
+                <field name="selling_price"/>
+                <field name="date_availability"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="estate_property_view_form" model="ir.ui.view">
+        <field name="name">estate_property_form</field>
+        <field name="model">estate.property</field>
+        <field name="arch" type="xml">
+            <form string="Estate Form View">
+                <sheet>
+                    <h1>
+                        <field name="name"/>
+                    </h1>
+                    <div class="d-flex justify-content-between">
+                        <div class="d-flex col">
+                            <group class="o_col">
+                                <field name="postcode"/>
+                                <field name="date_availability"/>
+                            </group>
+                        </div>
+                        <div class="d-flex col">
+                            <group class="o_col">
+                                <field name="expected_price"/>
+                                <field name="selling_price"/>
+                            </group>
+                        </div>
+                    </div>
```

Comment:
Isn't this similar to:
```suggestion
                    <group>
                        <group class="o_col">
                            <field name="postcode"/>
                            <field name="date_availability"/>
                        </group>
                        <group>
                            <field name="expected_price"/>
                            <field name="selling_price"/>
                        </group>
                    </group>
```

Let me know ?

Reply:
Yeah you're right. Group actually makes it more readable.